policies/versioning-policy.md

```diff
@@ -0,0 +1,133 @@
+# OpenSSL release versioning policy
+
+This document describes the release versioning scheme used by the OpenSSL
+project from version 3.0.0 onwards.  It also details the level of [ABI] and
+[API] compatibility each version represents.
+
+> _Note: All examples herein are illustrative and do not constitute part of the
+> versioning policy._
+
+The version scheme consists a triple of numbers:
+**[major]**.**[minor]**.**[patch]**.
+
+> For example, the version _3.0.1_ has a _major_ version of _3_, a _minor_
+> version of _0_ and a _patch_ version of _1_.
+
+This closely aligns with the expectations of users who are familiar
+with _semantic versioning_.  However, it is not quite _semantic versioning_
+because adopting _semantic versioning_ would mean changing our current
+Long Term Support ([LTS]) policies and practices.
+
+## Major release
+
+A major release is indicated by changing the _first digit_ of the version.
```

Comment:
Using "first digit", "second digit" and "final digit" precludes versions such as `3.2.10` and `3.10.0`. Would "number" work better?

Reply:
I've changed to "number" throughout and got the italics consistent.  Also changed a "last digit" to "final number".